plugins/redis/README.md

```diff
@@ -114,7 +113,8 @@ myRedis.SetString('myKey', 'bar', 1);
 myRedis.GetString('myKey', 0);//returns 'foo'
 myRedis.GetString('myKey', 1);//returns 'bar'
 ```
-*Note:* that the default database is 0.
+
+*Note:* that the default database is 0. The maximum number of databases allowed by **Redis** is 32768 (+ve range of int32).
```

Comment:
I wouldn't count that as pedantic, I'll change and push along with any other required changes.